### PR TITLE
Adding deprecated message

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1513

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: SharePoint/SharePointServer/administration/variations-overview.md [hunk 1]
```diff
@@ -21,7 +21,10 @@ description: "Learn about variations and the benefits and scenarios for using va
...
   
 This article contains an overview of the variations feature. It describes the elements of the variations feature; provides an overview of site, list, and page creation for variation sites; lists some limitations of variations; and describes scenarios for using variations in SharePoint Server. This article does not describe the tasks that are involved in planning a solution that uses variations. For information about how to plan to use variations in your solution, see [Plan for variations in SharePoint Server](plan-for-variations.md). This article also does not describe how to create variation labels and hierarchies. For information about how to create a variation site, see [Create a multi-language website](https://go.microsoft.com/fwlink/p/?LinkId=279696).
-  
+
+    > [!IMPORTANT]
+    > The Variations will remain supported but deprecated for the SharePoint Server 2019 release. For more information, see [What's deprecated or removed from SharePoint Server 2019](https://docs.microsoft.com/sharepoint/what-s-new/what-s-deprecated-or-removed-from-sharepoint-server-2019#variations)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/variations-overview.md]:
```suggestion
    > The variations will remain supported but deprecated for the SharePoint Server 2019 release. For more information, see [What's deprecated or removed from SharePoint Server 2019](https://docs.microsoft.com/sharepoint/what-s-new/what-s-deprecated-or-removed-from-sharepoint-server-2019#variations).
```
I agree Variations should be capitalized, but in this article it is not. So for the sake of consistency I suggest changing it to lower case.

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@JoanneHendrickson please review and merge

**Pull request merged**